Code/autopkglib/MunkiImporter.py

```diff
@@ -539,23 +577,44 @@ def main(self):
             return
 
         # copy pkg/dmg to repo
-        relative_path = self.copy_item_to_repo(pkginfo)
+        uploaded_pkgpath = munkiimportlib.copy_item_to_repo(
+            repo,
+            self.env["pkg_path"],
+            pkginfo.get("version"),
+            self.env.get("repo_subdirectory", ""),
+        )
         # adjust the installer_item_location to match the actual location
         # and name
-        pkginfo["installer_item_location"] = relative_path
+        pkginfo["installer_item_location"] = uploaded_pkgpath.partition("/")[2]
```

Comment:
This seems slightly risky - are there cases where this could cut improperly or paths could be longer than expected?